integration_test/agents/agents.go

```diff
@@ -932,3 +1030,27 @@ func installWindowsPackageFromGCS(ctx context.Context, logger *log.Logger, vm *g
 	}
 	return nil
 }
+
+func GetOtelConfigPath(imageSpec string) string {
+	isUAPPlugin := IsOpsAgentUAPPlugin()
```

Comment:
I think we can improve the if structure here by nesting UAP cases together.
if UAP {
...
}
<original code>